resolwe/toolkit/tools/parse_tabular_file.py

```diff
@@ -23,7 +21,7 @@ def main():
     args = parse_arguments()
 
     ext = os.path.splitext(args.input_file)[-1].lower()
-    with gzip.open(args.output_file, mode='wt') as outfile:
+    with gzip.open(args.output_file, mode='wt', encoding='utf-8') as outfile:
```

Comment:
The proper fix for using UTF-8 encoding by default is to [coerce the default `C` locale in the Docker container to `C.UTF-8`](https://www.python.org/dev/peps/pep-0538/).

Fedora will [properly coerce the `C` locale to `C.UTF-8` in version 26](https://fedoraproject.org/wiki/Changes/python3_c.utf-8_locale) which was released just today.

I'll prepare a PR that will update the `resolwe/base` image to Fedora 26.